### PR TITLE
Tevin Isaac-Sage-How to create a Token Staking Dapp

### PR DESCRIPTION
**Pull request by Tevin-Isaac**

### CODE REVIEW

**Comment by Tevin-Isaac** [comment]
> Thanks for submitting this tutorial for Celo SageBefore you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).
> 
> Please confirm the following before submitting your tutorial for review -
> 
> * [x]   My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
> * [x]   The title of the Pull Request matches the title of my tutorial.
> * [x]   I've followed the tutorial template structure.
> * [x]   I've proofread the tutorial for spelling and grammar mistakes.
> * [x]   I've thoroughly explained the code, with inline code comments and text explanations.
> * [x]   I've run a linter & prettifier to format my code blocks.
> * [x]   I've run the tutorial to test it and the code works.NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.Reviewers - [@joenyzio](https://github.com/joenyzio) & [@viral-sangani](https://github.com/viral-sangani)

**Review by Julius170 — COMMENTED** [review]
Hi @Tevin-Isaac You've got a really cool piece here, although needs a lot of corrections, Ive already left comments to guide you now and for future references, basically you can use pretifier next time to  help prevent most of the suggestive changes you'll need to make here

Also take note of the comments I made, the places they need to be effected at were more than the commented areas, so even if i didn't add a comment in some places to prevent too much redundancy check the rest of the article and make the changes based on the comments where necessary with or without the comments there.... can't wait to hear from you 🥂

**Review by Julius170 — COMMENTED** [review]
Hi @Tevin-Isaac You've got a really cool piece here, although needs a lot of corrections, Ive already left comments to guide you now and for future references, basically you can use pretifier next time to  help prevent most of the suggestive changes you'll need to make here

Also take note of the comments I made, the places they need to be effected at were more than the commented areas, so even if i didn't add a comment in some places to prevent too much redundancy check the rest of the article and make the changes based on the comments where necessary with or without the comments there.... can't wait to hear from you 🥂

**Review by Julius170 — COMMENTED** [review]
Hi @Tevin-Isaac You've got a really cool piece here, although needs a lot of corrections, Ive already left comments to guide you now and for future references, basically you can use pretifier next time to  help prevent most of the suggestive changes you'll need to make here

Also take note of the comments I made, the places they need to be effected at were more than the commented areas, so even if i didn't add a comment in some places to prevent too much redundancy check the rest of the article and make the changes based on the comments where necessary with or without the comments there.... can't wait to hear from you 🥂

File: blog/22-03-3023-nft-minter/how-to-build-an-nft-marketplace-with-nextjsipfsetherjs-and-redux.md
[the remark this answers is not in the document]
Comment (Tevin-Isaac) [reply blog/22-03-3023-nft-minter/how-to-build-an-nft-marketplace-with-nextjsipfsetherjs-and-redux.md:1]: oh it must have pushed coz i was using it to compare it was my previous submitted tutorial.

File: blog/19-04-2023-How to create a Token Staking Dapp/how-to-create-a-token-staking-dapp.md [hunk 1]
```diff
@@ -0,0 +1,2110 @@
...
+
+## Compile and Deploy
+Now that we are set with our project files we will now use Truffle to compile and deploy our smart contract for it to be able to work in the front end when we run our project locally.
+To do so lets first run this command:
+```
+truffle compile
+```
+When i run that command this is my output:
+```
+Compiling your contracts...
```
[the remark this answers is not in the document]
Comment (Tevin-Isaac) [reply blog/19-04-2023-How to create a Token Staking Dapp/how-to-create-a-token-staking-dapp.md:1954]: cool i was thinking of that as well until i saw let me just put them as code but noted.

File: blog/19-04-2023-How to create a Token Staking Dapp/how-to-create-a-token-staking-dapp.md [hunk 2]
```diff
@@ -0,0 +1,2110 @@
...
+> Final cost:          0.02956799 ETH
+
+```
+Now our migration files and our contracts have been deployed successfully and the transaction has been excecuted  you can even view it on the Celo Testnet block explorer.
+
+:::tip
+Incase you need to confirm the transaction hash   **[Celo Testnet Block Explorer](https://alfajores.celoscan.io)** you can view it here.
+:::
+
+For testing purposes of our smart contracts run this command below:
```
[the remark this answers is not in the document]
Comment (Tevin-Isaac) [reply blog/19-04-2023-How to create a Token Staking Dapp/how-to-create-a-token-staking-dapp.md:2058]: done added bash on all commands

**Comment by Tevin-Isaac** [comment]
i have done the necessary changes kindly check.

**Pull request closed without merging**